client/rpc/src/chain/mod.rs

```diff
@@ -275,6 +305,15 @@ impl<Block, Client> ChainApi<NumberFor<Block>, Block::Hash, Block::Header, Signe
 	fn unsubscribe_finalized_heads(&self, metadata: Option<Self::Metadata>, id: SubscriptionId) -> RpcResult<bool> {
 		self.backend.unsubscribe_finalized_heads(metadata, id)
 	}
+
+	// TODO: Figure out what `T` should be for `Subscriber`
```

Comment:
Should be all the finality data (so I guess header + justification). Might need a RPC-specific type to encapsulate that.